lib/hyacinth/adapters/digital_object_search_adapter/solr/document_generator.rb

```diff
@@ -48,11 +48,18 @@ def merge_title_fields_for(digital_object, solr_document = {})
         end
 
         def merge_title_fields_for!(digital_object, solr_document = {})
-          solr_document['sort_title_ssi'] = digital_object.generate_title(true)
-          indexable_title = digital_object.generate_title
-          solr_document['title_ss'] = indexable_title
-          add_keywords(indexable_title, solr_document)
-          add_titles(indexable_title, solr_document)
+          display_label = digital_object.generate_label
+          solr_document['displayLabel_ss'] = display_label
```

Comment:
```suggestion
          solr_document['display_label_ss'] = display_label
```
We've been going for snake case in solr keys to match the associated property names.